hi-ml-azure/src/health_azure/mlflow_utils.py

```diff
@@ -0,0 +1,48 @@
+#  ------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  ------------------------------------------------------------------------------------------
+"""
+Utility functions for interacting with mlflow runs
+"""
+from typing import Any, Dict, List
+from mlflow.client import MlflowClient
+from mlflow.entities import Run, Metric
+
+
+def get_mlflow_run(mlflow_client: MlflowClient, mlflow_run_id: str) -> Run:
```

Comment:
do we need a function for this? can't we jus use mlflow_client.get_run(mlflow_run_id) directly?
same for get_metric_from_mlflow_run and get_last_metrics_from_mlflow_run 
++reduces codecov complains

Reply:
These will go on to replace our functions for downloading runs from AML so yes, we do need these